test/message.spec.js

```diff
@@ -43,7 +56,7 @@ describe('BitswapMessage', () => {
     const raw = pbm.Message.encode({
       wantlist: {
         entries: [{
-          block: mh.toB58String(new Buffer('hello')),
+          block: new Buffer('hello'),
```

Comment:
`.toString('binary')`

avoid using a valid utf8 string here because otherwise the protobuf won't catch it. Put some random bytes